prisma/schema.prisma

```diff
@@ -3,6 +3,86 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
+  provider = "sqlite"
   url      = env("DATABASE_URL")
 }
+
+
+
+model Gestor {
+  id  String @id @default(uuid())
+  nome String
+  imagem String
+  email String @unique
+  area String
+  senha String   
+  
+  colaboradores Colaborador[]
+  indicadores Indicador[]
+}
+
+
+
+model Colaborador {
+  id  String @id @default(uuid())
+  nome  String // not null
+  email String @unique // not null
+  telefone  String @unique // not null
+  cargo String
+  data_admissao DateTime @default(now())
+  endereco  String
+  cep String
+  imagem  String
+
+  colaboradorIndicadores ColaboradorIndicador[]
+  notasMensais NotaMensalColaborador[]
+
+  // FK pra gestor
+  idGestor String
+  gestor Gestor @relation(fields: [idGestor], references: [id])
+}
+
+
+
+model Indicador {
+  id String @id @default(uuid())
+  nome String // not null
+  unidade_medida String
+  data_deadline DateTime
+  descricao String
+  colaboradorIndicadores ColaboradorIndicador[]
```

Comment:
Não consegui ter a perfeita compreensão deste atributo. Achei meio ambíguo a inclusão dele, pois em ColaboradorIndicador já possuímos a chave estrangeira que identifica o indicador a que cada linha se refere, mas acredito que são duas possibilidades para fazer essa identificação, embora que para manter um banco mais enxuto creio que devemos utilizar referências e não utilizar o objeto em si, em GDI vemos que essas abordagens são em paradigmas diferentes. Eu prefiro a abordagem de utilizar apenas a chave estrangeira, para ficarmos com o paradigma relacional mesmo.

Reply:
Tás falando do `colaboradorIndicadores ColaboradorIndicador[]`? Se for, é que precisa incluir isso no modelo, senão dá um erro e nem gera o BD. No SQL de fato não aparece atributo a mais

Reply:
Nooossa! Que estranho, mas então tá perfeito!